heartbeat/monitors/active/http/config.go

```diff
@@ -103,6 +114,20 @@ var defaultConfig = Config{
 	},
 }
 
+func (r *responseConfig) Validate() error {
+	switch strings.ToLower(r.IncludeBody) {
```

Comment:
Do you really want to do the strings to lower? I would enforce users to write it correctly.

Reply:
I mean, I don't see one way as being correct or another. Some people like typing in upper case. It's not a big deal to me however.

Reply:
Agree, just personal preference. Both options ok for me.